packages/jsii/lib/assembler.ts

```diff
@@ -22,13 +22,22 @@ const sortJson = require('sort-json');
 
 const LOG = log4js.getLogger('jsii/assembler');
 
+export interface AssemblerOptions {
+  /**
+   * Do not emit warnings for reserved words.
+   * @default false (warnings are emitted)
+   */
+  readonly reservedWordsWarningsDisabled?: boolean;
```

Comment:
Very wordy. Could shrink it down to `reservedWordWarns` which defaults to `true` or something like that.

Reply:
My (not so strong, but still) opinion on this subject is that optional `boolean` should default to `false`, not `true`. It's bugs waiting to happen otherwise as `null` and `undefined` are both falsey.

Reply:
I don't think optional booleans must default to `false` (especially given the double negative potential). In this specific case, however, since this is somewhat of an "escape hatch" I decided to be a bit less ergonomic (hence the long name and the false default).

Reply:
I believe you're going to apply this across the CDK, no? Which means this "escape hatch" is going to be used everywhere.

My opinion on this is not strong enough to prevent approval of this PR, but pointing out that this wordy flag is going to be everywhere.

Reply:
"everywhere" in the CDK v1.0. When we line up v2.0 we should re-enable and fix all the warnings.